samples/openthread/coprocessor/overlay-hci.conf

```
@@ -0,0 +1,40 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
```

```suggestion

CONFIG_RCP_SAMPLE_HCI=y
```

and in `Kconfig` for example:
```
config CONFIG_RCP_SAMPLE_HCI
	bool "Enable  Bluetooth HCI USB support for the coprocessor sample"
```